miete/models.py

```diff
@@ -31,19 +31,12 @@ class Meta:
         'Jahr der letzten Mieterhöhung', null=True, blank=True)
     vermieter = models.CharField(
         verbose_name='Vermieter', max_length=2, choices=VERMIETER_CHOICES, blank=True)
+    email = models.EmailField(
+        'E-Mail für Benachrichtigung über Ergebnis', blank=True)
```

Comment:
Einspruch! ich habe extra aus Datenschutzgründen email in eine eigene Tabelle ausgelagert, damit die Mieten Daten nicht unnötig mit Personen verknüpft werden können